components/Filter/styles.js

```diff
@@ -0,0 +1,28 @@
+import styled from 'styled-components';
+
+export const FilterContainer = styled.div`
+  background-color: #FFF;
+  border: 2px solid #000; /* Schwarzer Rand */
+  position: fixed;
+  display: flex;
+  color: #000;
+  padding: 20px;
+  margin-top: 120px;
+  flex-direction: column;
+  width: 100vw;
```

Comment:
This styling doesn't work right. I would delete this line. If you move your Filter into Header it will have the same width.